ledger/complete/wal/checkpointer.go

```diff
@@ -338,6 +367,21 @@ func LoadCheckpoint(filepath string) (*flattener.FlattenedForest, error) {
 		tries[i] = storableTrie
 	}
 
+	if version == VersionV3 {
```

Comment:
what about opt-in for future versions
```suggestion
	if version >= VersionV3 {
```

Reply:
We don't know what future versions will bring, let's keep it strict here